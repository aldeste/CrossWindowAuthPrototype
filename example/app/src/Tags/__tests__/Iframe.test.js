import Iframe from "../Iframe";
import renderer from "react-test-renderer";
import React from "react";
import "jest-styled-components";

describe("Iframe tag", () => {
  it("Should be defined", () => {
    expect(Iframe).toBeDefined();
  });

  it("Matches earlier configuration", () => {
    const tree = renderer.create(<Iframe />).toJSON();
    expect(tree).toMatchStyledComponentsSnapshot();
  });
});
