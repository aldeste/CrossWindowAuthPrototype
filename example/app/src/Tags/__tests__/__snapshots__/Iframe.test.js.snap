// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Iframe tag Matches earlier configuration 1`] = `
.HaLVb {
  border: 0;
  margin-left: calc(-50vw + 50%);
  margin-right: calc(-50vw + 50%);
  filter: invert();
}

<iframe
  className="Iframe__Iframe-hXkCjZ HaLVb"
/>
`;
